components/brave_wallet_ui/page/wallet_page.tsx

```diff
@@ -32,6 +32,7 @@ import { store, walletPageApiProxy } from './store'
 import walletDarkTheme from '../theme/wallet-dark'
 import walletLightTheme from '../theme/wallet-light'
 import 'emptykit.css'
+import '@brave/leo/tokens/css/variables.css'
```

Comment:
Hey, instead of importing Leo variables like this, we should add a
`<link rel="stylesheet" href="chrome://resources/brave/leo/css/variables.css">` to the relevant HTML file :smile: